iiwa_goal:
  x: -11.78233433590118
  y: 7.373602910182024
  yaw: 0.0
  z: 0.0
